store_app/views.py

```diff
@@ -45,3 +45,22 @@ def get_context_data(self, **kwargs):
         context['products_for_month'] = products_for_month
         context['products_for_year'] = products_for_year
         return context
+
+
+def edit_product(request, product_id):
+    product = get_object_or_404(Product, id=product_id)
+    if request.method == 'POST':
+
+        form = ProductEditForm(request.POST, request.FILES, instance=product)
```

Comment:
А тут `instance=product` ни к чему
```suggestion
        form = ProductEditForm(request.POST, request.FILES)
```